cosmos/dbt/selector.py

```diff
@@ -221,7 +253,16 @@ def filter_nodes(self, nodes: dict[str, DbtNode]) -> set[str]:
                         if config_selection_value == node.config.get(config_selection_key, "")
                     }
                 )
-
+            elif config_selection_key.startswith(CONFIG_META_PATH):
+                root_nodes.update(
+                    {
+                        node_id
+                        for node_id, node in nodes.items()
+                        if _check_nested_value_in_dict(node.config, f"{config_selection_key}:{config_selection_value}")
+                    }
+                )
+            else:
```

Comment:
It seems now we would still be logging a warning even if the `config_selection_key` is either `materialized` or `schema` whereas earlier that was not the case, no?

Reply:
Hm, I don't think this will be the case because of:
https://github.com/astronomer/astronomer-cosmos/blob/929e309d95bb9a8c6b859681231b486b7121217c/cosmos/dbt/selector.py#L245-L255